src/ops/element_binary.cc

```diff
@@ -97,6 +97,10 @@ Tensor FFModel::divide(const Tensor in1,
   return this->binary(OP_EW_DIV, in1, in2, inplace_a, name);
 }
 
+bool ElementBinaryParams::is_valid(const std::pair<ParallelTensorShape, ParallelTensorShape>&) const {
+  return true;
```

Comment:
Feel free to add some checks here on the shapes (i.e. for ElementBinary the two inputs should have the same shape).

Reply:
We'll do this in a later PR.